### DISCUSSION TITLE
saving the volume setting for each player and provider + display volume level

### DISCUSSION THREAD

**Discussion by darkman242** [music-assistant/support#3099] [Feature requests and ideas]
It would be nice if the selected volume could be saved for each player and provider. Radio provider, for example, is louder than spotify. when changing the provider, the volume must be adjusted for the same player.
It would also be useful to display the selected volume when the slider is pressed (similar to the HomeAssistant player display).

![image](https://github.com/user-attachments/assets/796b182e-0f51-4e86-92ef-67422fbb49bf)

**Comment by OzGav** [comment]
See if this is useful info https://music-assistant.io/faq/tech-info/#volume-normalization
